.travis.yml

```diff
@@ -5,6 +5,8 @@ rvm:
   - 2.5
   - 2.6
   - 2.7
+  - '3.0'
```

Comment:
This needs to be a string otherwise 3.0 resolves to 3 which will select the latest 3.x version of ruby.